src/components/NotFound.jsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './NotFound.less';
+import NotFoundSVG from '../img/not-found.svg';
+
+class NotFound extends React.Component {
+  render() {
+    return (
+      <div className='error-placeholder__error-msg'>
+        <h1>{'Oops! We can\'t seem to find the page you\'re looking for...'}</h1>
+        <p>
+        Either URL is not valid, or you do not have access.
```

Comment:
Could you add "this"? So that it says "Either this URL is not valid, or you do not have access."